src/olympia/addons/models.py

```diff
@@ -591,8 +591,9 @@ def initialize_addon_from_upload(cls, data, upload, channel, user):
         return addon
 
     @classmethod
-    def from_upload(cls, upload, platforms, channel=amo.RELEASE_CHANNEL_LISTED,
-                    parsed_data=None, user=None):
+    def from_upload(cls, upload, selected_apps,
+                    channel=amo.RELEASE_CHANNEL_LISTED, parsed_data=None,
+                    user=None):
         """
         Create an Addon instance, a Version and corresponding File(s) from a
         FileUpload, a list of platform ids, a channel id and the
```

Comment:
heads up: The docstring will need to be updated

Reply:
yeah, once I'm done with the first wave of changes I'll do a whole-code sweep for anything `platform` related I may have missed :)

Thanks!